payment-request/payment-response/helpers.js

```diff
@@ -1,5 +1,20 @@
 setup({ explicit_done: true, explicit_timeout: true });
 
+test(() => {
+  try {
+    const details = {
+      total: {
+        label: "",
+        amount: { currency: "USD", value: "1.0" },
+      },
+    };
+    const request = new PaymentRequest([{ supportedMethods: "basic-card" }]);
```

Comment:
I think the "details" should be passed into the constructor.

Reply:
d'oh! good catch.